src/functions/get-validation-history.ts

```diff
@@ -1,68 +1,104 @@
-import { APIGatewayProxyEvent, Context } from 'aws-lambda';
+import { APIGatewayProxyEvent, Context } from "aws-lambda";
 import {
   ValidationHistoryRequest,
   ValidationHistoryResponse,
   ValidationCount,
   ValidationAttempts,
-} from '../types/database';
-import { LambdaResponse } from '../types/response';
-import { LambdaResponses } from '../utils/lambda-responses';
-import { getValidationAttempts } from '../utils/database';
-import { checkValidColumnName, createSql } from '../utils/sql';
-import { Logger } from '../utils/logger';
+} from "../types/database";
+import { LambdaResponse } from "../types/response";
+import { LambdaResponses } from "../utils/lambda-responses";
+import { getValidationAttempts } from "../utils/database";
+import { checkValidColumnName, createSql } from "../utils/sql";
+import { Logger } from "../utils/logger";
+import { checker } from "../utils/cors-helper";
+import { StatusCodes } from "http-status-codes";
 
-export const getValidationHistory = async (event: APIGatewayProxyEvent, context: Context): Promise<LambdaResponse> => {
+export const getValidationHistory = async (
+  event: APIGatewayProxyEvent,
+  context: Context
+): Promise<LambdaResponse> => {
   const logger = new Logger(context);
   const params = event.queryStringParameters || {};
-
-  if (params.accountId && params.searchPhrase) {
-    return LambdaResponses.badRequest;
+  let origin = undefined;
+  if (event.headers.origin) {
+    origin = [event.headers.origin];
+  } else {
+    origin = [""];
   }
+  let allowed = "Denied";
```

Comment:
I prefer to move it to the const variable

Reply:
It's a let since it's being reassigned later.